odoo/addons/openupgrade_records/lib/compare.py

```diff
@@ -95,35 +95,42 @@ def report_generic(new, old, attrs, reprs):
                 text = "now required"
                 if new['req_default']:
                     text += ', default = %s' % new['req_default']
-                fieldprint(old, new, None, text, reprs)
+                fieldprint(old, new, '', text, reprs)
+        elif attr == 'stored':
+            if old[attr] != new[attr]:
+                if new['stored']:
+                    text = "now is stored"
```

Comment:
Slight preference for "now stored" (or else "is now stored", but not "now is stored").